chromium_src/components/translate/core/browser/translate_script.cc

```diff
@@ -9,6 +9,8 @@
 #include "base/strings/strcat.h"
 #include "base/strings/stringprintf.h"
 #include "base/threading/sequenced_task_runner_handle.h"
+#include "brave/common/brave_services_key.h"
```

Comment:
pls add gn deps for these files into `brave_components_translate_core_common_public_deps`

Reply:
In fact it's not possible without moving the files:
```
ERROR Dependency cycle:
  //components/translate/core/common:common ->
  //brave/common:common ->
  //chrome/common:common ->
  //components/translate/core/common:common
```
Let's leave this as is until we fix the deps here.